dotnet-client-libraries/TestDataGenerationWithAutoFixture/TestDataGenerationWithAutoFixture/Employee.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace TestDataGenerationWithAutoFixture
+{
+    public class Employee
+    {
+        public string FirstName { get; set; }
+        public string? LastName { get; set; }
+        public int? Age { get; set; }
+        public decimal Salary { get; set; }
+
+        public Employee
+            (string firstName,
+            decimal salary)
```

Comment:
This constructor should just call into the other constructor (Constructor Chaining)